modules/installation-obtaining-installer.adoc

```diff
@@ -95,6 +95,12 @@ ifndef::ibm-z,private,vsphere[* You have a computer that runs Linux or macOS, wi
 
 ifndef::openshift-origin[]
 . Go to the link:https://console.redhat.com/openshift/install[Cluster Type] page on the {hybrid-console}. If you have a Red{nbsp}Hat account, log in with your credentials. If you do not, create an account.
+
+[TIP]
+====
+You can also download the binaries for a specific {product-title} release.
```

Comment:
do we also need the link https://mirror.openshift.com/pub/openshift-v4/clients/ocp/ here?